css/comp_lib_styles/components/button.css

```diff
@@ -0,0 +1,63 @@
+.btn {
+  padding: 0.5rem 1rem;
+  margin: 0.5rem 0;
```

Comment:
Just a suggestion : You can use px instead of rems for units less than 10px. Though this is not mandatory.